exercises/house/src/test/java/HouseTest.java

```diff
@@ -105,15 +78,7 @@ public void verseSeven() {
     @Ignore("Remove to run test")
     @Test
     public void verseEight() {
-        String expected =
-                "This is the man all tattered and torn\n" +
-                "that kissed the maiden all forlorn\n" +
-                "that milked the cow with the crumpled horn\n" +
-                "that tossed the dog\n" +
-                "that worried the cat\n" +
-                "that killed the rat\n" +
-                "that ate the malt\n" +
-                "that lay in the house that Jack built.";
+        String expected = "This is the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.";
```

Comment:
Some of these lines are a bit long - we should still be able to break them over a couple of lines. This will allow them to be read more easily and edited more easily too!

Reply:
That makes sense. Will make the changes.